### ISSUE TITLE
Missing parameter in can-override.js

### ISSUE THREAD

**Issue by wahler** [jakubpawlowicz/clean-css#567]
I'm having an error with the latest version 3.2.8.
At this point https://github.com/jakubpawlowicz/clean-css/blob/v3.2.8/lib/properties/can-override.js#L56 function "sameFunctionOrValue" is called with two parameters, but expects three, see https://github.com/jakubpawlowicz/clean-css/blob/v3.2.8/lib/properties/can-override.js#L73

This results in:
.../node_modules/gulp-minify-css/node_modules/clean-css/lib/properties/can-override.js:78
TypeError: Cannot call method 'areSameFunction' of undefined

**Comment by jakubpawlowicz** [comment]
Good catch, @wahler. Any chance you can share the piece of CSS it fails on? If not I'll try to figure out what it can be. I'll fix it on spot tomorrow.

**Comment by wahler** [comment]
Thanks for your quick response, @jakubpawlowicz 
property1 equals
```
{ components: [],
  dirty: false,
  hack: false,
  important: false,
  name: 'background-color',
  multiplex: false,
  position: 0,
  shorthand: false,
  unused: false,
  value: [ [ '-ms-linear-gradient(top,#efefef 0,#f8f8f8 100%)' ] ] }
```
and property2 equals
```
{ components: [],
  dirty: false,
  hack: false,
  important: false,
  name: 'background-color',
  multiplex: false,
  position: 0,
  shorthand: false,
  unused: false,
  value: [ [ 'linear-gradient(top,#efefef 0,#f8f8f8 100%)' ] ] }
```

Does that help?

**Comment by jakubpawlowicz** [comment]
Amazing! :-) Fix is coming.

**Comment by jakubpawlowicz** [comment]
@wahler Fixed and released in 3.2.9.
